frontend/www/js/omegaup/components/admin/Support.vue

```diff
@@ -181,40 +191,43 @@ export default class AdminSupport extends Vue {
   @Prop() username!: string;
   @Prop() verified!: boolean;
   @Prop() link!: string;
-  @Prop() lastLogin!: Date;
+  @Prop() lastLogin!: null | Date;
+  @Prop() birthDate!: null | Date;
 
   T = T;
   ui = ui;
+  time = time;
   email: null | string = null;
   newEmail: null | string = null;
 
   @Emit('search-email')
-  onSearchEmail(): void | string {
-    if (this.email == null) return;
+  onSearchEmail(): null | string {
+    if (this.email == null) return null;
     return this.email;
   }
 
   @Emit('update-email')
-  onUpdateEmail(): UpdateEmailRequest | void {
-    if (this.email == null || this.newEmail == null) return;
+  onUpdateEmail(): null | UpdateEmailRequest {
+    if (this.email == null || this.newEmail == null) return null;
     return { email: this.email, newEmail: this.newEmail };
   }
 
   @Emit('verify-user')
-  onVerifyUser(): void | string {
-    if (this.email == null) return;
+  onVerifyUser(): null | string {
+    if (this.email == null) return null;
     return this.email;
   }
 
   @Emit('generate-token')
-  onGenerateToken(): void | string {
-    if (this.email == null) return;
+  onGenerateToken(): null | string {
+    if (this.email == null) return null;
     return this.email;
   }
 
-  @Emit('reset')
-  onReset(): UpdateEmailRequest {
-    return { email: this.email, newEmail: this.newEmail };
+  onReset() {
+    this.email = null;
+    this.newEmail = null;
+    this.$emit('reset');
```

Comment:
```suggestion
  @Emit('reset');
  onReset() {
    this.email = null;
    this.newEmail = null;
```

sí estaba bien antes. un `@Emit` con _sólo `void`_ es legal: https://github.com/kaorun343/vue-property-decorator#Emit

lo que es ilegal es mezclar `void` con cualquier otro tipo.

Reply:
Listo